src/actions/index.js

```diff
@@ -0,0 +1,72 @@
+import axios from 'axios';
+
+const resourceId = '35de6bf8-b254-4025-89f5-da9eb6adf9a0';
+
+export function loadFilters(){
+  let sql = encodeURI(`SELECT "LEVEL_1_CATEGORY" as name, COUNT(*) as num FROM "${resourceId}" GROUP BY name ORDER BY name`);
+  let url = `https://catalogue.data.govt.nz/api/3/action/datastore_search_sql?sql=${sql}`;
+  return (dispatch) => {
+    return axios.get(url).then((response)=>{
+      dispatch(showFilters(response.data.result.records));
+    });
+  };
+}
+
+const fields = ()=> {
+  return 'LEVEL_1_CATEGORY,FSD_ID,LONGITUDE,LATITUDE,PROVIDER_NAME,PUBLISHED_CONTACT_EMAIL_1,PUBLISHED_PHONE_1,PROVIDER_CONTACT_AVAILABILITY,ORGANISATION_PURPOSE,PHYSICAL_ADDRESS,SERVICE_NAME,SERVICE_DETAIL,DELIVERY_METHODS,COST_TYPE,SERVICE_REFERRALS';
+};
+
+export function loadResults(category, keyword) {
+  
+  let url = `https://catalogue.data.govt.nz/api/3/action/datastore_search?resource_id=${resourceId}&fields=${fields()}`;
+  var query;
+  if (keyword.length > 2) {
```

Comment:
i'm thinking this `if` could refactor down a bit more to read like english
:elephant:

Reply:
@Br3nda good spotting, that was definitely a code smell. See what I've done now, I moved the conditionals into their own functions.